workflows/iuf/operations/managed-nodes/managed-nodes-rollout.yaml

```diff
@@ -169,10 +169,10 @@ spec:
           function main_sat_command {
               print_inputs_variables
               if [ -z "$managed_rollout_strategy" ] || [[ "$managed_rollout_strategy" == *"$JSONPATH"* ]] ; then
-                  echo "ERROR: No managed_rollout_strategy provided."
+                  echo "ERROR No managed_rollout_strategy provided. Cannot proceed further"
```

Comment:
You might add. "Please provide -mrs reboot or -mrs stage".  Note the CLI provides a default value so theoretically you should never get here.